openquake/hmtk/seismicity/declusterer/__init__.py

```diff
@@ -15,3 +15,9 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+from openquake.hmtk.seismicity.declusterer.dec_afteran import Afteran
+from openquake.hmtk.seismicity.declusterer.dec_gardner_knopoff import (
+    GardnerKnopoffType1)
+from openquake.hmtk.seismicity.declusterer.distance_time_windows import (
+    UhrhammerWindow, GardnerKnopoffWindow, GruenthalWindow)
```

Comment:
You need to add two comments
```
# noqa
```
after the imports so that tools like pyflakes will not complain. For the rest, LGTM